Makefile.am

```diff
@@ -366,7 +421,6 @@ nobase_include_HEADERS += \
 	valhalla/mjolnir/util.h \
 	valhalla/mjolnir/validatetransit.h
 libvalhalla_la_SOURCES += \
-	src/proto/transit.pb.cc \
```

Comment:
we only need to use this in a couple of programs so i thought we shouldnt make it part of the library